modules/gapi/include/opencv2/gapi/video.hpp

```diff
@@ -62,6 +62,60 @@ G_TYPED_KERNEL(GCalcOptFlowLKForPyr,
         return std::make_tuple(empty_array_desc(), empty_array_desc(), empty_array_desc());
     }
 };
+
+enum BackgroundSubtractorType
+{
+    TYPE_BS_MOG2,
+    TYPE_BS_KNN
+};
+
+/** @brief Structure for initialization of the Background Subtractor operation's parameters.*/
```

Comment:
`Structure for the Background Subtractor operation's initialization parameters.`

Reply:
Applied.